src/tablist.js

```diff
@@ -123,6 +125,16 @@ SideTabList.prototype = {
       return;
     }
     if (e.which == 2 && SideTab.isTabEvent(e, false)) {
```

Comment:
This is to prevent the autoscrolling feature right? I think we should add a comment to make that clearer.
Also, can we remove SideTab.isTabEvent() ?